examples/cp4ba/variables.tf

```diff
@@ -1,5 +1,8 @@
+variable "ibmcloud_api_key" {
+  description = "Enter your IBM API Cloud access key. Visit this link for more information: https://cloud.ibm.com/docs/account?topic=account-userapikey&interface=ui "
+}
 
-variable "cluster_name_or_id" {
+variable "cluster_id" {
   default     = ""
   description = "Enter your cluster id or name to install the Cloud Pak. Leave blank to provision a new Openshift cluster."
```

Comment:
This example won't provision a cluster.  Cluster id needs to be a required variable.  So remove the "leave blank to provision a new OpenShift Cluster"

Reply:
Fixed